content/docs/setup/kubernetes/helm-install/index.md

```diff
@@ -103,30 +93,16 @@ Choose this option if your cluster doesn't have [Tiller](https://github.com/kube
 
 1. Render and apply Istio's core components:
 
-    {{< text bash >}}
+    * For a production environment use the recommended Helm values or customize as needed:
 
+    {{< text bash >}}
     $ helm template $HOME/istio-fetch/istio --name istio --namespace istio-system | kubectl apply -f -
-
     {{< /text >}}
 
-1. Uninstall steps:
+    *  To try the `bookinfo` demo, use Helm values customized for the `bookinfo` application:
```

Comment:
This text and the corresponding command should be a in a call out. To learn more about callouts in Istio, visit: https://preliminary.istio.io/about/contribute/creating-and-editing-pages/#callouts
```suggestion
    {{< tip >}}
    To try the `bookinfo` demo, use the Helm values customized for the `bookinfo` application:
    {{< text bash >}}
    $ helm template $HOME/istio-fetch/istio --name istio --values install/kubernetes/helm/istio/values-istio-demo.yaml --namespace istio-system | kubectl apply -f -
    {{< /text >}}
    {{< /tip >}}
```
Please verify the content renders correctly. The nesting is theoretically possible but I have never done it nor know of a place where we already use it.

Reply:
I think we need to separate the eval mode from the prouction mode.